src/models/tools/tool-content-info.ts

```diff
@@ -18,17 +19,33 @@ export interface IDefaultContentOptions {
   unit?: UnitModelType;
 }
 
+type ToolComponentType = React.ComponentType<IToolTileProps>;
+
 export interface IToolContentInfo {
   id: string;
   tool: string;
-  titleBase?: string;
   modelClass: typeof ToolContentModel;
+  defaultContent: (options?: IDefaultContentOptions) => ToolContentModelType;
+  Component: ToolComponentType;
+  toolTileClass: string;
+  titleBase?: string;
   metadataClass?: typeof ToolMetadataModel;
   addSidecarNotes?: boolean;
   defaultHeight?: number;
   exportNonDefaultHeight?: boolean;
-  defaultContent: (options?: IDefaultContentOptions) => ToolContentModelType;
   snapshotPostProcessor?: ToolTileModelContentSnapshotPostProcessor;
+  /**
+   * If the tile component doesn't call ui.setSelectedTile itself, then it can
+   * add  tileHandlesSelection: true and the tool-tile wrapper will handle the
+   * selection instead
+   * I think the name of this property is referring to ToolTileComponent as the "tile".
+   * So a tool is saying I don't handle my selection let my "tile" do it for me.
+   * Currently this is used by the table and drawing tools.
+   *
+   * This approach was first added in the commit below, this helps clarify its purpose:
+   * https://github.com/concord-consortium/collaborative-learning/commit/d19b201dfd2c635aae2f30672c50610f90ba07a5
+   */
+  tileHandlesSelection?: boolean;
```

Comment:
I think if you rename this to something more appropriate, as suggested above, then this comment could be simplified.

Reply:
done